app/views/users/_follow_form.html.erb

```diff
@@ -0,0 +1,5 @@
+<% case current_user.following_status_for(@user) 
```

Comment:
create a helper method like this:
```
def following_status_partial_name(follower, followed)
  case follower.following_status_for(followed)
  when 'not_following' then 'follow'
  when 'pending' then 'cancel'
  when 'active' then 'unfollow'
  end
end
```

and here:
`<%= render "users/#{following_status_partial_name(current_user, @user)}" %>`